crates/sui-json-rpc/src/read_api.rs

```diff
@@ -239,48 +230,21 @@ impl SuiRpcModule for ReadApi {
 #[async_trait]
 impl RpcFullNodeReadApiServer for FullNodeApi {
     async fn dev_inspect_transaction(
-        &self,
-        tx_bytes: Base64,
-        epoch: Option<EpochId>,
-    ) -> RpcResult<DevInspectResults> {
-        let epoch = match epoch {
-            None => self.get_sui_system_state_object_epoch()?,
-            Some(n) => n,
-        };
-        let (txn_data, txn_digest) = get_transaction_data_and_digest(tx_bytes)?;
-        Ok(self
-            .state
-            .dev_inspect_transaction(txn_data, txn_digest, epoch)
-            .await?)
-    }
-
-    async fn dev_inspect_move_call(
         &self,
         sender_address: SuiAddress,
-        package_object_id: ObjectID,
-        module: String,
-        function: String,
-        type_arguments: Vec<SuiTypeTag>,
-        arguments: Vec<SuiJsonValue>,
+        tx_bytes: Base64,
```

Comment:
hmm I need to take a closer look at this - the problem is that on the client side there's no easy way to serialize data into `TransactionKind` - we only expose API for constructing `TransactionData`